RELEASES.md

```diff
@@ -1,3 +1,131 @@
+Version 1.4.0 (October 2015)
```

Comment:
I hadn't thought about the anchor. Interesting point. I'd be fine rethinking how it's presented.